backend/erc20.go

```diff
@@ -80,6 +80,24 @@ var erc20Tokens = []erc20Token{
 		unit:  "ZRX",
 		token: erc20.NewToken("0xe41d2489571d322189246dafa5ebde1f4699f498", 18),
 	},
+		{
+		code:  "eth-erc20-wbtc",
+		name:  "Wrapped BTC",
+		unit:  "WBTC",
+		token: erc20.NewToken("0x2260fac5e5542a773aa44fbcfedf7c193bc2c599", 8),
+	},
+	{
+		code:  "eth-erc20-paxg",
+		name:  "Pax Gold",
+		unit:  "PAXG",
+		token: erc20.NewToken("0x45804880De22913dAFE09f4980848ECE6EcbAf78", 18),
+	},
+	{
+		code:  "eth-erc20-dgx",
```

Comment:
Our product people asked to not include this token for now (needs more research), so please remove it for the time being. The other two are good to be included.

Reply:
done